docs/generated/packages/angular/documents/overview.md

```diff
@@ -27,6 +27,10 @@ yarn add -D @nx/angular
 npm install -D @nx/angular
 ```
 
+{% callout type="note" title="Keep Nx Package Versions In Sync" %}
+Make sure to install the `@nx/angular` version that matches the version of `nx` in your repository. If the version numbers get out of sync, you can encounter some difficult to debug errors. Run `nx report` to see a list of installed plugins. Use [`nx migrate`](/core-features/automate-updating-dependencies) to update plugin versions.
+{% /callout %}
```

Comment:
Do we need to call out here that if you already have a a higher version of one of the packages, you may need to run `nx migrate --from --to` or are we expecting the user to get that information from the migrate docs?